src/Role.php

```diff
@@ -37,7 +37,7 @@ protected function __construct(protected string $name, protected int $threshold,
      *
      * @see https://theupdateframework.github.io/specification/v1.0.20#document-formats
      */
-    public static function createFromMetadata(\ArrayObject $roleInfo, string $name): Role
+    public static function createFromMetadata(iterable $roleInfo, string $name): Role
```

Comment:
`@param \ArrayObject $roleInfo` still needs to be updated. PHPCS doesn't catch this?

Reply:
Fixed. I guess PHPStorm didn't catch it because the actual type hint is `iterable`, which `ArrayObject` falls into. I've changed them both to `array`.